src/Cli/dotnet/commands/dotnet-workload/install/IWorkloadInstaller.cs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+
+namespace Microsoft.DotNet.Workloads.Workload.Install
+{
+    internal interface IWorkloadInstaller
```

Comment:
@dsplaisted please review this interface. "WorkloadInstaller" is not a concept in the spec. And the name is too vague. We should come up with a better naming and coin that in the spec

Reply:
This was meant to act as the IAL but I figured that wasn't a very descriptive name in this context. Definitely open to other suggestions, though.